karma.conf.js

```diff
@@ -123,6 +127,19 @@ module.exports = function(config) {
       });
     }
 
+    if (options['native-iterator']) {
+      customLaunchers = {
+        SL_Chrome: {
+          base: 'SauceLabs',
+          browserName: 'chrome'
+        },
+        SL_Firefox: {
+          base: 'SauceLabs',
+          browserName: 'firefox'
+        }
```

Comment:
I think you only need one launcher called `Evergreen` with `{ firefox: 'dev', chrome: 'dev'` }`?